README.md

```diff
@@ -858,8 +858,7 @@ here's the gist of it:
 
 **DO NOT merge from master to your feature branch; rebase.**
 
-Note that all dcrdata.org community and team members are expected to adhere to
-the code of conduct, described in the [CODE_OF_CONDUCT](docs/CODE_OF_CONDUCT.md)
+Note that all dcrdata.org community and team members are expected to follow the Code Contribution Guidelines, described in the [Code Contribution Guidelines](https://github.com/decred/dcrd/blob/master/docs/code_contribution_guidelines.md)
 file. These guidelines are generally not a challenge for decent humans.
```

Comment:
Since the CoC is, gone and the Code Contribution Guidelines do not touch on acceptable behavior in the same sense that the CoC did, the sentence "These guidelines are generally not a challenge for decent humans." is no longer applicable.

Reply:
I removed it.